src/groups.ts

```diff
@@ -34,7 +50,9 @@ export function isSelectedViaGroup(
     .find((groupId) => appState.selectedGroupIds[groupId]);
 }
 
-export function getSelectedGroupIds(appState: AppState): GroupId[] {
+export function getSelectedGroupIds(
+  appState: AppState,
+): ExcalidrawElement["groupIds"] {
```

Comment:
nit: not too sure about this change since i don't think the return value and the value of `groupIds` necessarily represent the same type. with `getSelectedGroupIds()` order doesn't matter so it really should return a `Set`, whereas order does matter in `groupIds`...

with that said this is a super nit, feel free to keep it if you want.

Reply:
You're right. I made the refactor so that people can easily find the docs for it, but it doesn't make sense here.